<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d1_Screen_9</class>
 <widget class="QWidget" name="d1_Screen_9">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
	<widget class="QWidget" name="d1_9_widgetBackground" native="true">
     <property name="styleSheet">
	  <string notr="true">#d1_9_widgetBackground{
	border-image: url(:/d1_noItem.png);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,3">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>177</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
	   <widget class="QWidget" name="d1_9_widgetContent" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
		  <layout class="QHBoxLayout" name="d1_9_hlTopWidget" stretch="10,1">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
			<layout class="QVBoxLayout" name="d1_9_vlText">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="d1_9_lblInfo">
               <property name="palette">
                <palette>
                 <active>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>81</red>
                     <green>17</green>
                     <blue>68</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </active>
                 <inactive>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>81</red>
                     <green>17</green>
                     <blue>68</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </inactive>
                 <disabled>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="96">
                     <red>164</red>
                     <green>166</green>
                     <blue>168</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </disabled>
                </palette>
               </property>
               <property name="font">
                <font>
				 <family>Baloo Cyrillic</family>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Возврат на карту</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignHCenter|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item alignment="Qt::AlignRight|Qt::AlignTop">
			<widget class="QPushButton" name="d1_9_btnClose">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
			  <string notr="true">#d1_9_btnClose{
	border-image: url(:/d1_btnClose.png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="d1_9_lblText_1">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>238</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>238</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="96">
                 <red>164</red>
                 <green>166</green>
                 <blue>168</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
			 <family>Baloo Cyrillic</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Внимание, возврат на карту обычно
происходит мгновенно,
но время может быть увеличено банком.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="d1_9_lblText_2">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>81</red>
                 <green>17</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>81</red>
                 <green>17</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="96">
                 <red>164</red>
                 <green>166</green>
                 <blue>168</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
			 <family>Baloo Cyrillic</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Для возврата средств приложите карту, с
которой была оплата.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
